gulpfile.js

```diff
@@ -941,65 +941,77 @@ function getRCBranchName() {
 };
 
 // Recompile and push to origin.
-gulp.task('git-recompile', gulp.series([
-    'git-sync-develop',
-    function(done) {
-      var branchName = getRebuildBranchName();
-      console.log('make-rebuild-branch: creating branch ' + branchName);
-      execSync('git checkout -b ' + branchName, { stdio: 'inherit' });
-      done();
-    },
-    'build',
-    'typings',
-    function(done) {
-      console.log('push-rebuild-branch: committing rebuild');
-      execSync('git commit -am "Rebuild"', { stdio: 'inherit' });
-      var branchName = getRebuildBranchName();
-      execSync('git push origin ' + branchName, { stdio: 'inherit' });
-      console.log('Branch ' + branchName + ' pushed to GitHub.');
-      console.log('Next step: create a pull request against develop.');
-      done();
+const recompile = gulp.series(
```

Comment:
I would prefer if we prefaced all the git related things with git, ``gitRecompile`` for this as a form of grouping.

Reply:
Discussed offline--they're still grouped in the exports.